docs/sphinx/manuals/roc_recv.rst

```diff
@@ -122,17 +118,23 @@ Listen on two ports on a particular interface:
 
     $ roc-recv -vv -s rtp+rs8m:192.168.0.3:10001 -r rs8m:192.168.0.3:10002
 
-Listen on two Reed-Solomon ports, two LDPC ports, and one bare RTP port:
+Listen on two ports on a particular multicast interface (but not join to a multicast group):
+
+.. code::
+
+    $ roc-recv -vv -s rtp+rs8m:225.1.2.3:10001 -r rs8m:225.1.2.3:10002
+
+Listen on two ports on a particular multicast interface and join to a multicast group on all interfaces:
 
 .. code::
 
-    $ roc-recv -vv -s rtp+rs8m::10001 -r rs8m::10002 -s rtp+ldpc::10003 -r ldpc::10004 -s rtp::10005
+    $ roc-recv -vv -s rtp+rs8m:225.1.2.3:10001 -r rs8m:225.1.2.3:10002 --miface 0.0.0.0
 
-Listen on two ports on all IPv4 interfaces and on two ports on all IPv6 interfaces:
+Listen on two ports on a particular multicast interface and join to a multicast group on a particular interface:
 
 .. code::
 
-    $ roc-recv -vv -s rtp+rs8m::10001 -r rs8m::10002 -s rtp+rs8m:[::]:10001 -r rs8m:[::]:10002
+    $ roc-recv -vv -s rtp+rs8m:225.1.2.3:10001 -r rs8m:225.1.2.3:10002 --miface 192.185.12.3
```

Comment:
I suggest to use `192.168.0.3` in line with other examples :)

Reply:
Done